practice-one/src/assets/css/footer.css

```diff
@@ -1,23 +1,18 @@
 .footer-section {
   background-color: var(--black-color);
   padding-top: 105px;
-  padding-left: 147px;
-  padding-right: 151px;
   padding-bottom: 91px;
 }
+
 .footer-container {
```

Comment:
The container class is used to wrap the content of all sections in a web. It should be a standard class.